src/api/app/models/staging_workflow.rb

```diff
@@ -20,10 +20,12 @@ def ignored_requests
 
   private
 
-  def init_staging_projects
-    return unless new_record?
+  def create_staging_projects
     ['A', 'B'].each do |letter|
-      staging_projects << Project.find_or_initialize_by(name: "#{project.name}:Staging:#{letter}")
+      p = Project.find_or_initialize_by(name: "#{project.name}:Staging:#{letter}")
```

Comment:
The name of a variable, function, or class, should answer all the big questions. It should tell you why it exists, what it does, and how it is used. If a name requires a comment, then the name does not reveal its intent.

Use intention revealing names as a tip @Ana06:

- Use pronounceable names
- Use searchable names
- Pick one word per concept

I recommend you a real good book about the theme: https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882  :satisfied: :satisfied: :satisfied: :grinning: :grinning: :grinning:  :heartbeat: :heartbeat: :heartbeat: :heartbeat: :heartbeat:

Reply:
you may want to read [my previous comment](https://github.com/openSUSE/open-build-service/pull/6193#discussion_r231198841). Also, do not forget that this is preventing us to work in many of the sprints cards.

Reply:
@Ana06 i don't see how writing short name variables make us faster :smirk: :smirk: :smirk:

Reply:
@Ana06 ```staging_project``` would be a good option :grimacing: :grimacing: :grimacing: :grimacing: :grimacing: